app/Services/Campaign/Import/Mappers/GalleryMapper.php

```diff
@@ -45,6 +45,8 @@ public function prepare(): self
     public function import(): void
     {
         $this->image = new Image();
+        //Need to set the id before saving otherwise it crashes
+        $this->image->id = Str::uuid();
```

Comment:
Remove this and call the ->save before calling the Mapping helper so that it doesn't crash. We are now using Str::orderedUuid because it's better for performance than Str::uuid